.tekton/retag-pipeline.yaml

```diff
@@ -76,27 +62,13 @@ spec:
     type: string
 
   results:
-  - description: ""
-    name: IMAGE_URL
-    value: $(tasks.retag-image.results.IMAGE_URL)
-  - description: ""
-    name: IMAGE_DIGEST
-    value: $(tasks.retag-image.results.IMAGE_DIGEST)
+  # IMAGE_URL and IMAGE_DIGEST must not be declared here because Tekton Chains will overwrite the original pipeline
+  # information linked to the image with this pipeline's info, and it will most certainly fail EC checks.
```

Comment:
You're declaring `RESULTING_URL` [there](https://github.com/stackrox/stackrox/pull/13309/files#diff-43b8ad4bdcbc5bcac7b8ff04b2a82b1ee9e890ce930422626f039738acc9a6b6R33), but it's missing here.
Might be useful.

Reply:
I thought it's redundant since `RESULTING_REF` has both "url" and digest but I'm also ok to make results identical with the task for consistency. Updated.